ydb/core/scheme/scheme_tablecell.h

```diff
@@ -120,6 +120,26 @@ struct TCell {
     const char* InlineData() const                  { Y_VERIFY_DEBUG(!IsInline_); return Ptr; }
     const char* Data() const                        { Y_VERIFY_DEBUG(!IsInline_); return Ptr; }
 #endif
+
+    void CopyData(char * dst) const {
+        if (CanInline(DataSize_)) {
```

Comment:
It should be `IsInline_` not `CanInline`. Then you should handle fixed size cases (including 0), and offload to memcpy everything else, using `Data()` instead of `Ptr`. I'd like code to be robust to unrelated changes.

Reply:
Updated to use `IsInline_`. Althought usage of `Data()` will produce additional branches.